src/modules/simulation/gz_bridge/GZBridge.hpp

```diff
@@ -94,7 +95,8 @@ class GZBridge : public ModuleBase<GZBridge>, public ModuleParams, public px4::S
 
 	void clockCallback(const gz::msgs::Clock &clock);
 
-	//void airpressureCallback(const gz::msgs::FluidPressure &air_pressure);
+	// void airspeedCallback(const gz::msgs::AirSpeedSensor &air_pressure);
+	void airpressureCallback(const gz::msgs::FluidPressure &air_pressure);
```

Comment:
Would it be possible to change the naming to something less confusing with the airspeed (e.g. `barometerCallback` ?)

Reply:
https://github.com/PX4/PX4-Autopilot/pull/21176/commits/e0de6492a0bb545b661adcd0234d21856aa25c26